osi_vehicle.proto

```diff
@@ -0,0 +1,829 @@
+syntax = "proto2";
+
+option optimize_for = SPEED;
+
+import "osi_version.proto";
+import "osi_common.proto";
+import "osi_object.proto";
+import "osi_occupant.proto";
+
+package osi3;
+
+//
+// \brief The vehicle-message is a deeper description of a vehicle.
+// Consists of different messages categorizing the vehicle in:
+// Vehicle-Basics, Vehicle-Kinematics, Vehicle-Powertrain, Vehicle-SteeringWheel, Vehicle-Wheels,
+// Vehicle-Light-State, Vehicle-Automated-Driving-Functions.
+// As it is an description of the whole vehicle (focus on cars) it can be used as interface for various reasons, 
+// e.g. to the vehicle-model for kinematics calculation or even for the visualisation in a graphic-engine.
+//
+message Vehicle
+{
+    //
+    // The interface version used by the sender (i.e. the simulation
+    // environment).
+    //
+    optional InterfaceVersion version = 1;
```

Comment:
I would move this into a root message like TrafficUpdate or similar and use OSI_Vehicle as a class

Reply:
Done.